src/Controls/src/Core/Compatibility/Handlers/Windows/FrameRenderer.cs

```diff
@@ -79,50 +76,21 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			{
 				UpdateCornerRadius();
 			}
-			else if (e.PropertyName == Frame.PaddingProperty.PropertyName)
-			{
-				UpdatePadding();
```

Comment:
Do we know if there's an existing test that covers this on WinUI or do we need to add an additional one?

Reply:
Good point! Test added :)